packages/lib/src/components/Doku/Doku.tsx

```diff
@@ -17,9 +17,9 @@ export class DokuElement extends UIElement {
      */
     formatData() {
         return {
+            ...this.state.data,
```

Comment:
Since we're refactoring this, could we change this to be more explicit on the fields that are returned?